src/app/business/block/file-share-detail/file-share-detail.component.ts

```diff
@@ -352,11 +352,22 @@ export class FileShareDetailComponent implements OnInit{
         this.SnapshotService.deleteSnapshot(id).subscribe((res)=>{
             this.getSnapshots(this.fromFileShareId);
             this.showCreateSnapshot = true;
+            this.msgs = [];
+            this.msgs.push({severity: 'success', summary: 'Success', detail: "Fileshare snapshot will be deleted shortly."});
```

Comment:
Do we update the real success information later...? How the user will know whether it is really deleted or not.

Reply:
The message is will be deleted shortly as the API is async. When the list is refreshed again the snapshot may not be visible. Sometimes the snapshot is deleted immediately and the list is refreshed before showing the user.

The dashboard works in a pull mode. So there is no way to update it dynamically.